chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
-
-Find your \textbf{home folder}. On MacOS, this will be a folder with your username.
-On Windows, this will be something like ``This PC''. (It is never your desktop.)
-Nearly everything we talk about will assume you are starting from here.
+  \index{password protection}
+All machines should have \textbf{hard disk encryption} enabled.
+  \index{encryption}
+Disk encryption is built-in on most modern operating systems;
+the service is currently called BitLocker on Windows or FileVault on MacOS.
+Disk encryption prevents your files from ever being accessed
+without first entering the system password.
+This is different from file-level encryption,
+which makes individual files unreadable without a specific key.
+(We will address that in more detail later.)
+As with all critical passwords, your system password should be strong,
+memorable, and backed up in a separate secure location.
+
+Make sure your computer is backed up to prevent information loss.
+  \index{backup}
+Follow the \textbf{3-2-1 rule}: maintain 3 copies of all critical data,
+on at least 2 different hardware devices you have access to,
+with 1 offsite storage method.\sidenote{
+  \url{https://www.backblaze.com/blog/the-3-2-1-backup-strategy/}}
+One reasonable setup is having your primary computer,
+a local hard drive managed with a tool like Time Machine
+(alternatively, a fully synced secondary computer),
+and either a remote copy maintained by a cloud backup service
+or all original files stored on a remote server.
+Dropbox and other synced files count only as local copies and never as remote backups,
+because other users can alter or delete them.
```

Comment:
* _Time Machine_ is a specific Mac tool. We can't introduce that without a link or without a Windows alternative.
* What do you mean by a local hard drive here? A second drive on your machine?

I think we can say the same thing in a way that does not introduce new software or terminology.

_One example of this setup is having one copy on your primary computer, one copy on an external hard drive stored in a safe place, and one copy in the cloud. In this case, Dropbox and other automatic file sync services does not count as your cloud copy, as other users can alter or delete them, unless you create a specific folder for this purpose that is not shared with anyone else._

Reply:
7595fad5aa947bf20ac7ba38cd78f515d9f3ecb2